### ISSUE TITLE
Logits defined as softmax output in A3C example

### ISSUE THREAD

**Issue by danijar** [ppwwyyxx/tensorpack#197]
In [train-atari.py line 107](https://github.com/ppwwyyxx/tensorpack/blob/master/examples/A3C-Gym/train-atari.py#L107) the logits are defined as softmax activation of the policy:

```python
self.logits = tf.nn.softmax(policy, name='logits')
```

However, the `policy` tensor already holds the logits, or unnormalized log probabilities. The softmax activation then computes the actual probabilities by applying the exponential function and normalizing. If I read the code correctly, it's just a wrong naming and not a bug in the implementation though.

**Comment by ppwwyyxx** [comment]
Right. I'll use a better name.
